src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-tts.ts

```diff
@@ -0,0 +1,109 @@
+import { css, CSSResultGroup, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import { SchemaUnion } from "../../../../components/ha-form/types";
+import { AssistPipeline } from "../../../../data/assist_pipeline";
+import { HomeAssistant } from "../../../../types";
+
+@customElement("assist-pipeline-detail-tts")
+export class AssistPipelineDetailTTS extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public data?: Partial<AssistPipeline>;
+
+  @property() public error?: Record<string, string>;
+
+  private _schema = memoizeOne(
+    (language?: string) =>
+      [
+        {
+          name: "",
+          type: "grid",
+          schema: [
+            {
+              name: "tts_engine",
+              selector: {
+                tts: {
+                  language,
+                },
+              },
+            },
+
+            {
+              name: "tts_language",
+              selector: {
+                text: {},
+              },
+            },
+            {
+              name: "tts_voice",
+              selector: {
+                text: {},
+              },
+            },
+          ] as const,
+        },
+      ] as const
+  );
+
+  private _computeLabel = (
+    schema: SchemaUnion<ReturnType<typeof this._schema>>
+  ): string =>
+    this.hass.localize(
+      `ui.panel.config.voice_assistants.assistants.pipeline.detail.form.${schema.name}`
+    );
+
+  protected render() {
+    return html`
+      <div class="section">
+        <div class="intro">
+          <h3>Text to speech</h3>
```

Comment:
We are going to follow the Microsoft style guideline and write it with dashes.
```suggestion
          <h3>Text-to-speech</h3>
```